bin/sized_list/prog1_syn.result

```diff
@@ -0,0 +1,4 @@
+generated error  fail  pass / total     time test name
+[2K
[ ]     0     0     0     0 / 20000     0.0s prog1_syn[2K
[[32;1m✓[0m] 20200     0     0 10152 / 20000     0.0s prog1_syn
+================================================================================
+[32;1msuccess[0m (ran 1 tests)
```

Comment:
I'm just looking through these numbers quickly to make a graph. Does this mean that our synthesized generators don't produce fully valid outputs?